api/migrations/token_blacklist_index_analysis.md

```
@@ -0,0 +1,102 @@
+# Token Blacklist Index Analysis
+
+## Current Index Configuration
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)